# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/d3d12/src/command_allocator.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/d3d12/src/command_allocator.rs b/third_party/rust/d3d12/src/command_allocator.rs
--- a/third_party/rust/d3d12/src/command_allocator.rs
+++ b/third_party/rust/d3d12/src/command_allocator.rs
@@ -1,14 +1,14 @@
-//! Command Allocator
-
-use crate::com::WeakPtr;
-use winapi::um::d3d12;
-
-pub type CommandAllocator = WeakPtr<d3d12::ID3D12CommandAllocator>;
-
-impl CommandAllocator {
-    pub fn reset(&self) {
-        unsafe {
-            self.Reset();
-        }
-    }
-}
+//! Command Allocator
+
+use crate::com::WeakPtr;
+use winapi::um::d3d12;
+
+pub type CommandAllocator = WeakPtr<d3d12::ID3D12CommandAllocator>;
+
+impl CommandAllocator {
+    pub fn reset(&self) {
+        unsafe {
+            self.Reset();
+        }
+    }
+}